src/cluster_info.rs

```diff
@@ -502,7 +503,19 @@ impl ClusterInfo {
                 br_idx
             );
 
-            orders.push((window_l[w_idx].data.clone(), &broadcast_table[br_idx]));
+            // Broadcast the last tick to everyone on the network so it doesn't get dropped
+            // (Need to maximize probability the next leader in line sees this handoff tick
+            // despite packet drops)
+            if idx == received_index - 1 && contains_last_tick {
+                // If we see a tick at max_tick_height, then we know it must be the last
+                // Blob in the window, at index == received_index. There cannot be an entry
+                // that got sent after the last tick, guaranteed by the PohService).
+                let all: Vec<&NodeInfo> = broadcast_table.iter().collect();
+                assert!(window_l[w_idx].data.is_some());
+                orders.push((window_l[w_idx].data.clone(), all));
+            } else {
+                orders.push((window_l[w_idx].data.clone(), vec![&broadcast_table[br_idx]]));
```

Comment:
Move this to after the if-else: `orders.push((window_l[w_idx].data.clone(), all));`?